viite-UI/src/view/JunctionEditForm.js

```diff
@@ -0,0 +1,154 @@
+(function (root) {
+    root.JunctionEditForm = function (backend) {
+        var svgJunction =
```

Comment:
svg pictures are already defined. I think you can use them like this : 

`> <object type="image/svg+xml" data="images/**filename**.svg>`

Reply:
Fixed and pushed to Git.